docs/developer-docs/latest/setup-deployment-guides/typescript.md

```diff
@@ -95,3 +125,116 @@ New plugins can be generated following the [plugins development documentation](/
 ::: note
 It is not necessary to repeat the `yarn install` or `npm run install` command after the initial installation. The `yarn build` or `npm run build` command is necessary to implement any plugin development that affects the admin panel.
 :::
+
+## Start Strapi programmatically
+
+Instantiating Strapi programmatically in a Typescript project requires additional configurations to load everything correctly. The primary difference for TypeScript programatic use is that the codebase and compiled code are stored in separate directories, whereas the same directory is used to read and write in native JavaScript.
+
+### Understand programatic use
+
+When instantiating Strapi programmatically using the default export of `@strapi/strapi`, different parameters such as the `app` directory and the `dist` directory can be passed. The `app` directory is the project root directory and `dist` is a subdirectory of `app`. The `dist` directory represents the compiled project, with the same directory structure as the `app` directory, and the `app` directory represents the codebase (TypeScript or JavaScript). The main differences are:
```

Comment:
This section (lines 135-139) looks a bit heavy IMHO.

What do you think about simplifying the presentation to make it easier to digest the content?

I'd use a table, for instance with 2 columns at least: parameter name (`app` or `dist`), and description. 
Maybe the description could be separated into 2 columns, like target directory and description/explanations.

If mentioning the `develop`-thingy is important (line 139), it could be included in a caution callout for instance.

What do you think?